quantity-pickers/src/test/kotlin/com/decathlon/vitamin/compose/quantity/pickers/VitaminQuantityPickersPrimaryTest.kt

```diff
@@ -0,0 +1,148 @@
+package com.decathlon.vitamin.compose.quantity.pickers
+
+import android.annotation.SuppressLint
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.Scaffold
+import androidx.compose.material.Text
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import app.cash.paparazzi.Paparazzi
+import com.decathlon.vitamin.compose.foundation.VitaminTheme
+import com.decathlon.vitamin.compose.quantity.pickers.utils.Theme
+import com.decathlon.vitamin.compose.quantity.pickers.utils.Variant
+import com.google.testing.junit.testparameterinjector.TestParameter
+import com.google.testing.junit.testparameterinjector.TestParameterInjector
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(TestParameterInjector::class)
+internal class VitaminQuantityPickersPrimaryTest(
```

Comment:
Check test results, helper text should be red

Reply:
Done